ajax/libs/jquery.basictable/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery.basictable",
+  "version": "1.0.6",
+  "filename": "jquery.basictable.min.js",
+  "homepage": "https://www.jerrylow.com/basictable/demo/",
+  "description": "Basic Table jQuery plugin for simple responsive tables.",
+  "keywords": [
+    "responsive",
+    "table",
+    "jquery"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jerrylow/basictable.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jerrylow/basictable.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*basictable.*+(js|css)"
```

Comment:
.map files could appear in the future. Please use `*basictable.*+(js|css|map)` instead.